optimism/src/mips/witness.rs

```diff
@@ -361,7 +379,7 @@ impl<Fp: Field> Env<Fp> {
                 .unwrap_or_else(|| "n/a".to_string());
 
             info!(
-                "processing step {} pc {:#010x} insn {:#010x} ips {:.2} page {} mem {} name {}",
+                "processing step={} pc={:#010x} insn={:#010x} ips={:.2} page={} mem={} name={}",
```

Comment:
The formatting here was the one that `cannon` uses, in order to ease the comparison between the outputs of the 2 VMs. Has `cannon`'s formatting changed?